client/lib/plans/features-list.js

```diff
@@ -616,14 +616,14 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_EMAIL_LIVE_CHAT_SUPPORT ]: {
 		getSlug: () => constants.FEATURE_EMAIL_LIVE_CHAT_SUPPORT,
-		getTitle: () => i18n.translate( 'Email & Live Chat Support' ),
+		getTitle: () => i18n.translate( 'Email & live chat support' ),
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **43** times:
`translate( 'Email & Live Chat Support' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/card-headings-form-labels-sentence-case)